packages/devtools-network-console/src/reducers/websocket/index.ts

```diff
@@ -33,70 +33,61 @@ export default function reduceWebsocket(collection: WS_State = DEFAULT_WS_STATE,
     if (!action.requestId) {
         return collection;
     }
-    // TODO: make switch statement
-    if (action.type === 'REQUEST_WEBSOCKET_CONNECTED') {
-        const reqId = action.requestId;
-        let state = collection.get(reqId);
-        if (!state) {
-            state = DEFAULT_WS_CONNECTION;
-        }
-        state = {
-            ...state,
-            connected: true,
-            messages: state.messages.add({
-                direction: 'status',
-                content: 'Connected',
-            })
-        };
-        return collection.set(reqId, state);
+    const reqId = action.requestId;
```

Comment:
oh, you switched to `switch` here. 👍

Reply:
Yep! I'll merge this into the first PR and then resolve that comment